src/Parser.php

```diff
@@ -447,14 +458,6 @@ private function validateAndGetNextTaggedNodeType(): string
     protected function parseStep()
     {
         $token = $this->expectTokenType('Step');
-        \assert(\is_string($token['value']));
-        \assert(\array_key_exists('keyword_type', $token));
-        \assert(\array_key_exists('text', $token));
-
-        $keyword = $token['value'];
-        $keywordType = $token['keyword_type'];
-        $text = trim($token['text']);
-        $line = $token['line'];
 
```

Comment:
I'd prefer not to have this variable inlining in the same commit diff as everything else.

I also think we've lost something here - `$token['value']` must always be present for a step token. Casting to a string, instead of the previous `assert(is_string` means a null will now be happily accepted when that should be invalid?

Reply:
Done: https://github.com/Behat/Gherkin/pull/352